ui/src/pages/QuickSetup/QuickSetup.tsx

```diff
@@ -17,25 +17,34 @@ import { WellKnownPartiesProvider } from '@daml/hub-react/lib';
 import { ledgerId, isHubDeployment } from '../../config';
 
 import Credentials, { computeCredentials } from '../../Credentials';
-import { retrieveParties } from '../../Parties';
+import { retrieveParties, retrieveUserParties } from '../../Parties';
 
 import { ArrowLeftIcon, ArrowRightIcon } from '../../icons/icons';
 
 import AddPartiesPage from './AddPartiesPage';
 import ReviewPage from './ReviewPage';
-import FinishPage from './FinishPage';
-import paths from '../../paths';
+import LoginPage from './LoginPage';
 import Widget from '../../components/Widget/Widget';
+import AssignRolesPage from './AssignRolesPage';
+import ProvideServicesPage from './ProvideServicesPage';
+import CreateAccountPage from './CreateAccountPage';
+import AssignProvidersPage from './AssignProvidersPage';
 
 export enum MenuItems {
   ADD_PARTIES = 'add-parties',
+  ASSIGN_PROVIDERS = 'assign-providers',
```

Comment:
Would calling this `Configure providers` instead of `Assign providers` make more sense?

Reply:
Oh, Will do!!